addons/video/src/presenter.js

```diff
@@ -1528,7 +1568,8 @@ function Addonvideo_create() {
         presenter.currentMovie = state.currentMovie;
         presenter.reload();
 
-        $(presenter.videoObject).on('canplay', function onVideoCanPlay() {
+        var eventType = presenter.isHLS() ? 'loadedmetadata' : 'canplay';
+        $(presenter.videoObject).on(eventType, function onVideoCanPlay() {
```

Comment:
Czy nie powinno być 
$(this).off(eventType);
zamiast 
$(this).off('canplay');
w 1578 linii ?